File: src/main/java/com/github/dylmeadows/eontimer/model/TimerSettingsConstants.java
package com.github.dylmeadows.eontimer.model;

import lombok.experimental.UtilityClass;

@UtilityClass
@SuppressWarnings({"unused", "WeakerAccess"})
public class TimerSettingsConstants {
    public final Console DEFAULT_CONSOLE = Console.NDS;
    public final boolean DEFAULT_PRECISION_CALIBRATION_MODE = false;
    public final int DEFAULT_REFRESH_INTERVAL = 8;
    public final int DEFAULT_MINIMUM_LENGTH = 14000;
}

File: src/main/kotlin/io/github/dylmeadows/eontimer/model/settings/TimerSettingsConstants.kt
@file:JvmName("TimerSettingsConstants")

package io.github.dylmeadows.eontimer.model.settings

object TimerSettingsConstants {
    @JvmField
    val DEFAULT_CONSOLE = Console.NDS

    const val DEFAULT_PRECISION_CALIBRATION_MODE = false
    const val DEFAULT_REFRESH_INTERVAL = 8L
    const val MINIMUM_LENGTH = 14000
}